Makefile

```diff
@@ -10,11 +10,14 @@ all: clean test
 
 clean:
 	$(PYTHON) setup.py clean
-	rm -rf dist
+	rm -rf dist build bin
 
+bin:
+	mkdir -p $@
+	PYTHONPATH=bin:$(PYTHONPATH) python setup.py develop --install-dir $@
 
-test-code:
-	$(NOSETESTS) -s -v $(MODULE)
+test-code: bin
+	PATH=bin:$(PATH) PYTHONPATH=bin:$(PYTHONPATH) $(NOSETESTS) -s -v $(MODULE)
```

Comment:
I have merged so I could easily test on windows VM we have -- the answer is "nope".  for some reason make just stalls if ran within bash, and make test makes mkdir puke if ran straight in far cmdline... but oh well ;)